cloud/compose.go

```diff
@@ -143,3 +143,19 @@ func (c Client) ComposeWait(id string, timeout, interval time.Duration) (aborted
 		}
 	}
 }
+
+// ListComposes returns status of all of the cloud composes on the server
+func (c Client) ListComposes() ([]ComposeInfo, error) {
```

Comment:
I just had a quick look at the `ComposeInfo` type and it does not map all [available properties](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/osbuild/osbuild-composer/refs/heads/main/internal/cloudapi/v2/openapi.v2.yml#operation/getComposeList). Probably because you don't need them (yet), right?
(like `image_status` or `href`)

Reply:
Right, the general idea with weldr-client is to *try* not to be tightly coupled to the server API, except where it really has to.